### ISSUE TITLE
Move the split button from the tabs panel to the tab

### ISSUE THREAD

**Issue by n7trd** [zed-industries/feedback#735]
### Check for existing issues

- [X] Completed

### Is your feature request related to a problem?

Not a problem but definitely an enhancement

### Describe the solution you'd like

Currently the "split button" is on the right side of the tabs.
While it looks like some kind of global action that affects the
overall tabs the action only applies to the currently open tab like
the close button only applies to a specific tab.

It would therefore move the button next to the close button.

**Pro:**
- It's more clear for the user what action affects what. 
- Its works the same way we are used to, if we work with windows
- I can hover over other tabs/file the are not in focus and split them 
- (optional )You gain some space for an "additional action button (...)" for real global actions, like close all tabs and so on :-)

**Con:** 
- I didn't see other editor do this but they are all bad design anyway :-)

Optional: Don't show a context menu with all split options if the user clicks on the button, just split it to the right site. That's what most dev do anyway. The option to split in an other direction is in the global menu anyway. Optionally there could add a setting to change the split direction.

Below: 
Mockup (mouse-over-state) with optional "additional action button "

### If applicable, add mockups / screenshots to help present your vision of the feature

![image](https://user-images.githubusercontent.com/44614967/203871726-924e5fc2-99c3-4a95-9717-fdb78a273be9.png)

**Comment by JosephTLyons** [comment]
I don't have a strong opinion on this, but I'm sure the team does, so I'll tag them in.

cc: @iamnbutler 

I think your list of pros and cons makes sense, but I think the last pro (gaining additional space) comes with a con: you lose space in the tabs.  Or you make each tab a tiny bit bigger and lose the global space you saved.

**Comment by n7trd** [comment]
> i think the last pro (gaining additional space) comes with a con: you lose space in the tabs.

Yes, you are right. Overall there would be less horizontal space for the tabs. That's definitely something that should be added to the cons list

**Comment by n7trd** [comment]
An other option would be to show a context menu* with split options, if the user right clicks on a specific tab.
Pro: The tabs don't need additional space for the split button & you don't need to decide on a specific default split option.
Yes, it is less discoverable but it's a ui behavior that is common enough.

*could be used for additional actions that affects the the selected tab. VSCode has a context menu for actions like this. Unfortunately it is cluttered with to many actions and has some actions that affects other tabs which is a really bad idea, in my view.

**Comment by JosephTLyons** [comment]
> An other option would be to show a context menu* with split options, if the user right clicks on a specific tab.
> 
> Pro: The tabs don't need additional space for the split button & you don't need to decide on a specific default split option.
> 
> Yes, it is less discoverable but it's a ui behavior that is common enough.
> 
> 
> 
> *could be used for additional actions that affects the the selected tab. VSCode has a context menu for actions like this. Unfortunately it is cluttered with to many actions and has some actions that affects other tabs which is a really bad idea, in my view.

Just wanted to mention that I don't think adding the icon and making the tab a little wider is a big deal, I just noting it.  It probably makes sense to do that.
